doc/index.rst

```diff
@@ -1,23 +1,39 @@
 .. _index:
 
-Welcome to nRF Connect SDK - Amazon Sidewalk
-############################################
+Welcome to the nRF Connect SDK - Amazon Sidewalk
+################################################
 
-Amazon Sidewalk is a shared network that allows your devices to work better at your home and outside of it.
-This documentation provides instructions on how to correctly set up Amazon Sidewalk from the perspective of the `nRF Connect SDK`_.
+Amazon Sidewalk is a shared network designed to provide a stable and reliable connection to your devices allowing them to work better at your home and outside of it.
+Every Ring and Echo device can act as a gateway device, meaning it can share a portion of internet bandwidth providing the connection and services to Sidewalk end devices.
```

Comment:
Not every Ring and Echo device can be a Sidewalk gateway - there is a list in Amazon doc with supported devices https://docs.sidewalk.amazon/introduction/sidewalk-gateways.html

Reply:
added the reference